runtime/src/lib.rs

```diff
@@ -1145,6 +1166,7 @@ construct_runtime!(
         MiningClaimsHardware: mining_claims_hardware::{Module, Call, Storage, Event<T>},
         MiningExecutionToken: mining_execution_token::{Module, Call, Storage, Event<T>},
         ExchangeRate: exchange_rate::{Module, Call, Storage, Event<T>},
+        Assets: pallet_assets::{Module, Call, Storage, Event<T>},
```

Comment:
i think this should be listed immediately below Tips instead of at the end

Reply:
Done